docs/source/workflows/predictor_reports.rst

```diff
@@ -13,56 +15,55 @@ Status can be one of:
 -  ``ERROR`` An error was thrown while generating the report.
 -  ``OK`` The report was generated successfully and the results are ready.
 
-Results of the report are stored as a dictionary in the ``json`` member variable.
-For the :class:`~citrine.informatics.predictors.SimpleMLPredictor`, this dictionary contains one key: ``important_features``.
-Feature importances are computed for each output descriptor.
-Important features are the input features that the machine learning model depends on most strongly.
-These importances can be useful in gaining some insight into how the machine learning model is making its predictions.
+Results of the report are in the ``descriptors`` and ``model_summaries`` attributes.
```

Comment:
It might be nice to include some json snippets of the sub-reports inline in this paragraph.

Reply:
I don't follow. What sub-reports?

Reply:
Feature importance and model settings.

Reply:
But aren't we trying to move away from talking about the raw JSON and focus on the objects that are created?